en-US/firefox/whatsnew_57.lang

```diff
@@ -0,0 +1,25 @@
+## NOTE: Demo page at https://www-demo3.allizom.org/firefox/57.0/whatsnew/
```

Comment:
This is not the DEMO server, we need to push craig to have one before we send out files for translation